bin/fin

```diff
@@ -167,6 +167,7 @@ URL_ADDONS_HOSTING="https://raw.githubusercontent.com"
 URL_ADDONS_REPO="$URL_ADDONS_HOSTING/docksal/addons"
 # Remove URL_REPO_ADDONS in Nov 2018
 URL_REPO_ADDONS="https://raw.githubusercontent.com/docksal/addons"
```

Comment:
This variable is unused now

Reply:
I can remove it, I was leaving it there because I know you said remove in Nov '18